From edd6e10a4510c9d715359f539afbf63240cf40d0 Mon Sep 17 00:00:00 2001
From: Marco Tomaschett <marco@entekadesign.com>
Date: Fri, 22 Nov 2024 13:08:06 -0500
Subject: [PATCH] hwdb: add support for PineTab2 to 60-sensor.hwdb (#35304)

Add accelerometer support for PineTab2

(cherry picked from commit bc4a027f9cb66d56fd1305f41bed25751305fc27)

Resolves: RHEL-49810
---
 hwdb.d/60-sensor.hwdb | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index 28d23da631..3c4bb81671 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -953,6 +953,15 @@ sensor:modalias:acpi:MXC6655*:dmi:*:svnDefaultstring*:pnP612F:*
 sensor:modalias:acpi:SMO8500*:dmi:*:svnPEAQ:pnPEAQPMMC1010MD99187:*
  ACCEL_MOUNT_MATRIX=-1, 0, 0; 0, 1, 0; 0, 0, 1
 
+#########################################
+# Pine64
+#########################################
+
+# PineTab2
+
+sensor:modalias:of:NaccelerometerT_null_Csilan,sc7a20:*
+ ACCEL_MOUNT_MATRIX=0, 0, -1; 1, 0, 0; 0, -1, 0
+
 #########################################
 # Pipo
 #########################################
